### PR TITLE
Migrate spring data cosmosdb sample

### PR DESCRIPTION
**Pull request by yiliuTo**
This pr migrates a sample of spring-data-cosmosdb to sdk/cosmos/azure-spring-data-cosmosdb/samples.

### CODE REVIEW

**Comment by yiliuTo** [comment]
Due to the parent of sample project is `org.springframework.boot: spring-boot-starter-parent`, currently no verion tags are added in the pom.xml.

**Comment by yiliuTo** [comment]
Hi @kushagraThapar , thanks for reminding me, I will rebase from master and re-push this PR after the src code PR gets merged

File: sdk/cosmos/azure-spring-data-cosmosdb/samples/.gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
```
Comment (saragluna) [inline sdk/cosmos/azure-spring-data-cosmosdb/samples/.gitignore:15]: There's no need for a gitignore file here.

File: sdk/cosmos/azure-spring-data-cosmosdb/samples/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
```
Comment (saragluna) [inline sdk/cosmos/azure-spring-data-cosmosdb/samples/LICENSE:16]: Same here.

**Comment by yiliuTo** [comment]
Close this PR for the commit history hash is different from the master

**Pull request closed without merging**